docs/guide/robots.md

```diff
@@ -54,6 +54,7 @@ Please [contact us](https://cyberbotics.com/#contact) if you would like to see y
 | ![icon.png](images/robots/robotis-op2/icon.png)      | [Robotis OP2](robotis-op2.md)             | Robotis              | *Small size humanoid*             |
 | ![icon.png](images/robots/robotis-op3/icon.png)      | [Robotis OP3](robotis-op3.md)             | Robotis              | *Small size humanoid*             |
 | ![icon.png](images/robots/salamander/icon.png)       | [Salamander](salamander.md)               | EPFL / BioRob        | *Bio-inspired robot*              |
+| ![icon.png](images/robots/scaraEpsonT6/icon.png)          | [ScaraT6](scara-epson-t6.md)              | Epson                | *SCARA robot*                     |
```

Comment:
This line should be aligned as the rest: [raw text](https://raw.githubusercontent.com/cyberbotics/webots/53c51a5930d4572acd25cb39eb665e002ae5da30/docs/guide/robots.md)

 (might as well fix the one of the Ned robot as well)